src/Stratis.Features.FederatedPeg/Wallet/FederationWalletManager.cs

```diff
@@ -470,12 +470,7 @@ public bool ProcessTransaction(Transaction transaction, int? blockHeight = null,
                 }
 
                 // Extract the withdrawal from the transaction (if any).
-                IWithdrawal withdrawal = null;
-                if (isDistribution)
-                    withdrawal = this.withdrawalExtractor.ExtractDistributionWithdrawal(transaction, blockHash, blockHeight ?? 0);
-                else
-                    withdrawal = this.withdrawalExtractor.ExtractWithdrawalFromTransaction(transaction, blockHash, blockHeight ?? 0);
-
+                IWithdrawal withdrawal = this.withdrawalExtractor.ExtractWithdrawalFromTransaction(transaction, blockHash, blockHeight ?? 0);
```

Comment:
Is `ExtractDistributionWithdrawal` still being used? If not, perhaps remove it.

Reply:
I will remove it 👍